psi4/driver/driver_cbs.py

```diff
@@ -1995,17 +2424,17 @@ def _cbs_gufunc(func, total_method_name, **kwargs):
         stage['basis'] = basis_list[0]
         if 'scf_scheme' in kwargs:
             stage['scheme'] = kwargs.pop('scf_scheme')
-        stage['stage'] = "scf"
-        stage['treatment'] = "scf"
+        #stage['stage'] = "scf"
```

Comment:
Why comment these out?

Reply:
So "stage" is just a print label for the tables in the output. The filling-in of default values is handled by the `_validate_cbs_inputs()` function.

And "treatment" used to determine which extrapolation scheme is used, along with an user-supplied "scheme". It was super-confusing to have these set by default, and the expansion of possible "treatments" from "scf" and "corl" to "fctl" and "dh" does not help. Now the key (which can still be user-supplied) is consumed by the validator to determine the appropriate scheme.

The "scheme" preference setting for each stage is now:
- either user-supplied via setting "scheme" directly
- or user-supplied via setting "treatment" and using the default table (`_get_default_xtpl()`)
- or a fallback setting using default table and the position of the stage in the overall recipe.